mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -23,6 +23,27 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
+  /** param for elastic net regularization parameter */
+  val alphaParam: DoubleParam = new DoubleParam(
```

Comment:
I would move it to `LinearRegression` and rename it to `alpha`. This is not a shared param.